media/js/base/mozilla-client.js

```diff
@@ -27,8 +100,20 @@ if (typeof Mozilla === 'undefined') {
     };
 
     /**
+     * Get the user's Firefox major version number. 0 will be returned on non-Firefox browsers.
+     *
+     * @private
+     * @param  {String} ua - browser's user agent string, navigator.userAgent is used if not specified
+     * @return {Number} major version number
+     */
+    Client._getFirefoxMajorVersion = function (ua) {
+        return parseFloat(Client._getFirefoxVersion(ua));
```

Comment:
Technically, won't this return more than just the major version number? e.g. `Won't this return more than the major version number alone? e.g. `Mozilla.Client._getFirefoxMajorVersion("Mozilla/5.0 (Macintosh; Intel Mac OS X 10.10; rv:44.1) Gecko/20100101 Firefox/44.1");`